docs/evaluate/development-production-features/multi-team.mdx

```diff
@@ -0,0 +1,93 @@
+As teams grow and applications evolve, modular architectures and seamless collaboration become critical. 
+With multiple teams collaborating, each responsible for their slice of the application, they want to build, operate, and deliver autonomously while allowing their slice to be used by others.
+
+As applications grow in complexity, managing Workflows across multiple teams and Namespaces is increasingly challenging. 
+Existing solutions for modularizing a Temporal Application face several issues, including tight coupling between Workflows that complicates management, limited modularity across Namespaces, security risks due to broad access permissions, complexity in managing cross-boundary interactions, and a lack of flexibility in adapting applications as they grow, often resulting in downtime or rework.
+
+Temporal Nexus offers a powerful solution by providing a modular architecture that supports autonomous team collaboration and a clean separation of concerns - while maintaining strong integration across the organization.
+This ensures that applications remain scalable, secure, and easy to manage across multiple Namespaces and regions.
+
+**Key Capabilities:**
+- **Modular Architecture:** Break down applications into manageable pieces that can be developed, maintained, and scaled independently.
+- **Autonomous Collaboration:** Enable teams to work independently while maintaining secure and efficient integration with other teams.
+- **Clean Separation of Concerns:** Keep different parts of the application isolated and secure, reducing complexity and enhancing manageability.
+- **Scalability and Flexibility:** Support cross-namespace and cross-region Workflows, ensuring that your application can grow and adapt as your organization evolves.
+- **Enhanced Security:** Leverage mTLS communication, Namespace allowlists, and strict access controls to protect sensitive operations and ensure compliance.
+
+## Core Components of Temporal Nexus
+
+Temporal Nexus supports improved modularity and multi-team collaboration, by allowing teams to define a clean Nexus Service API contract for other teams to use, that abstract underlying Temporal primitives, like Workflows.
+
+Nexus Services are exposed from a Nexus Endpoint that is created in the Nexus API Registry, along with service documentation that enable others to use the Nexus Services provided by a Nexus Endpoint. This enables teams to compose Nexus Services into their workflows, both within and across namespace boundaries.
+
+Nexus is designed to connect durable executions across team and namespace boundaries with an integrated Temporal SDK experience, built-in Nexus Machinery that processes Nexus RPC calls on your behalf, and promotes a more modular architecture for sharing a subset of your team’s capabilities via a well-defined Nexus Service API contract.
+
+**Nexus API Registry:** A `Nexus API Registry` is a list of Nexus `Endpoints` available within a Temporal environment. 
```

Comment:
Is the official term for this API Registry? I think in Cloud we call this the Endpoint Registry

Reply:
@prasek Currently I have API Registry in the glossary.

Reply:
Yes Nexus API Registry b/c over time it will be more than that, Endpoints are just the first thing

See [current definition](https://github.com/temporalio/documentation/pull/3046/files#diff-c42a05dfff7873800ca86ea805b61b7a52aefb8f9462d0e4adb5753a037d4bfaR323)